misc/anonymization.md

```diff
@@ -92,3 +92,34 @@ Computed Properties:
 
 
 <sub>*\* Features that are marked ~~strikethrough~~ are not implemented yet.*</sub>
+
+
+## Anonymization
```

Comment:
@viliambalaz Akoze aky ma vyznam nadpis "Anonymization" v dokumente o anonymizacii? Sak rozmyslaj

Reply:
@viliambalaz V ostatnych `misc/*.md` suboroch zvykneme mat 2nd level nadpisy ako `## Views`, `## Wizards`, `## Events` a podobne a az pod nimi konkretne viewy, wizardy, eventy ako 3rd level nadpisy. 

Tak aj tu analogicky by sa zisiel 2nd level nadpis nieco ako `## Commands` a az potom 3rd level nadpis ```### `attachment_anonymization` ```

Reply:
Teraz som si vsimol ze nie je dodrzany nas "standard", dva prazdne riadky medzi nadpismi (ako napr. v README.md). Ale aby to bolo rovnako v ramci suboru, tak som zatial nechal jeden prazdny riadok. Mozem to zeditovat v novom PR (aj v ostatnych `.md` suboroch).

Reply:
@viliambalaz Ano, pls vzdy ked najdes, ze niekde nedodrzujeme nejaku convention, tak v separatnom PR to oprav v celej codebase. Diki.